CHANGES.md

```diff
@@ -1,22 +1,28 @@
 Change Log
 ==========
 
+### 1.40 - 2017-12-01
+
+* Added ability to support touch event in Imagery Layers Split demo application. [#5948](https://github.com/AnalyticalGraphicsInc/cesium/pull/5948)
+
 ### 1.39 - 2017-11-01
 
-* Added support for right-to-left language detection in labels, currently Hebrew and Arabic are supported. To enable it, set `Cesium.Label.enableRightToLeftDetection = true` at the beginning of your application. [#5771](https://github.com/AnalyticalGraphicsInc/cesium/pull/5771)
```

Comment:
@ggetz are you sure this CHANGES.md diff is OK?

Reply:
@pjcozzi I'm not sure what went wrong with this diff, but I just checked and CHANGES is correct in master

Reply:
OK, thanks!